app/answerOperations.js

```diff
@@ -0,0 +1,58 @@
+var Question = require('../models/question');
+var Answer = require('../models/answer');
+var textUtils = require('../utils/textutils');
+var tokenUtils = require('../utils/tokenutils');
+var Constants = require('../constants');
+
+module.exports = function () {
+    return setupFunctions();
```

Comment:
Is there any advantage of having the ``setupFunctions()``? Instead, couldnt we move it in like this:

```javascript
module.exports = function(){
    var postAnswer = function postAnswer ( req, res ) { 
       \\ ...
    };
    \\ ...
    return { postAnswer: postAnswer };
}
```

There may be an advantage to your method that I am missing. But right off the bat, its not quite sure what the separate function gains us.